### DISCUSSION TITLE
How to use render hooks?

### DISCUSSION THREAD

**Discussion by mloru** [filamentphp/filament#3478] [Help]
I'd want to display some information in the pages' footer.

As stated [in the docs](https://filamentphp.com/docs/2.x/admin/appearance#render-hooks) I've created a new `partials.footer-content` view, I've added 

```
Filament::registerRenderHook(
    'footer.start',
    fn (): View => view('partials.footer-content'),
);
```

to the `boot()` method in my `AppServiceProvider` and I've set the config entry `filament.layout.footer.should_show_logo` to `true` (not sure if this is mandatory).

Still I cannot see any changes in my template. Am I misunderstanding something?

**Comment by danharrin** [comment]
You're not misunderstanding anything, so I am confused why that doesnt work 🤔
